atd-vze/src/views/Crashes/CrashChangeLog.js

```diff
@@ -118,37 +118,47 @@ class CrashChangeLog extends Component {
     // Define a function to pass to JSON.stringify which will skip over
     // the __typename key in the JS object being stringified.
     const stringifyReplacer = (key, value) => {
-      if (key === '__typename') {
+      if (key === "__typename") {
         return undefined;
       }
       return value;
-    }
+    };
 
     // For each entry created in the diff array, generate an HTML table row.
-    let modalItems = diff.map(item => {
+    let modalItems = diff.map((item, i) => {
```

Comment:
this fixes a key warning. Index is okay here since the order of these items won't change after initial render.